oap-ws-sso-api/src/main/java/oap/ws/sso/interceptor/SecurityInterceptor.java

```diff
@@ -77,14 +79,16 @@ public Optional<Response> before( InvocationContext context ) {
         Optional<User> u = context.session.get( SESSION_USER_KEY );
         if( u.isEmpty() ) return Optional.of( new Response( UNAUTHORIZED ) );
 
-        Optional<String> realm = context.getParameter( wss.get().realm() );
+        Optional<String> realm =
+            wss.get().realm().equals( SYSTEM) ? Optional.of( SYSTEM ) : context.getParameter( wss.get().realm() );
         if( realm.isEmpty() ) return Optional.of( new Response( FORBIDDEN, "realm is not passed" ) );
 
         Optional<String> role = u.flatMap( user -> user.getRole( realm.get() ) );
-        if( role.isEmpty() )
-            return Optional.of( new Response( FORBIDDEN, "user doesn't have access to realm " + realm.get() ) );
+        if( role.isEmpty() && u.map( usr -> usr.getRole( SYSTEM ) ).isEmpty() ) return Optional.of( new Response( FORBIDDEN, "user doesn't have access to realm " + realm.get() ) );
 
-        if( roles.granted( role.get(), wss.get().permissions() ) ) return Optional.empty();
+        if( ( !realm.get().equals( SYSTEM ) && u.map( usr -> usr.getRole( SYSTEM ) ).isPresent() )
```

Comment:
Same thing as previous. Role will be provided from getRole and this role should be checked for permissions.